inventory.py

```diff
@@ -1,97 +1,128 @@
+import textwrap
+
 class Order:
     def __init__(self, purchase = {}) -> None:
+        # Unclear naming here. I would assume add_purchase would replace this,
+        # not update this. Even renaming it to `purchases` would be clearer

```

Comment:
That makes sense, this should just be all purchases passed to the instance, not an expanding list.

Reply:
No, the list can continue to expand just like you've done it, you just need to rename it. Something like `self.purchases`, that way it's clear that `add_purchase` augments `self.purchases`. If it's `self.purchase`, I would assume that `add_purchases` replaces it.

Reply:
That makes sense.